### ISSUE TITLE
PermissionError: [WinError 5] Access is denied: 'er-patcher-tmp\\eldenring.exe'

### ISSUE THREAD

**Issue by PatrickJnr** [gurrgur/er-patcher#45]
with `python er-patcher --all --executable launch_elden_ring_seamlesscoop.exe -- %command%`

I get

```
E:\Steam\steamapps\common\ELDEN RING\Game\er-patcher:137: DeprecationWarning: pathlib.Path.link_to() is deprecated and is scheduled for removal in Python 3.12. Use pathlib.Path.hardlink_to() instead.
  f.link_to(game_dir_patched / f)
Traceback (most recent call last):
  File "E:\Steam\steamapps\common\ELDEN RING\Game\er-patcher", line 145, in <module>
    rmtree(game_dir_patched)
  File "C:\Users\patri\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 749, in rmtree
    return _rmtree_unsafe(path, onerror)
  File "C:\Users\patri\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 619, in _rmtree_unsafe
    onerror(os.unlink, fullname, sys.exc_info())
  File "C:\Users\patri\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 617, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: 'er-patcher-tmp\\eldenring.exe'
```

**Comment by myholymoonlight** [comment]
Deprecation Warning: The first part of the error message is a DeprecationWarning indicating that the link_to() method of the pathlib.Path object is being deprecated and is scheduled for removal in Python 3.12. The warning suggests that you use the hardlink_to() method instead to avoid this issue.

Permission Error: The second part of the error message is a PermissionError stating that access is denied to the file er-patcher-tmp\\eldenring.exe. This error suggests that you do not have sufficient permissions to modify the file. To resolve this, you should try running the command as an administrator or make sure that you have full permissions to the folder and file in question.

Here are some steps you can try to resolve these issues:

Replace all instances of link_to() with hardlink_to() in the er-patcher script.

Make sure that you are running the command as an administrator or have sufficient permissions to modify the file. You may need to adjust the file permissions to allow modifications.

**Comment by gurrgur** [comment]
This is fixed by #78
